fv3core/stencils/d_sw.py

```diff
@@ -359,63 +367,21 @@ def ke_horizontal_vorticity(
         )
 
 
-def initialize_heat_source(heat_source, diss_est):
-    heat_source[grid().compute_interface()] = 0
-    diss_est[grid().compute_interface()] = 0
-
-
-def heat_from_damping(
-    ub,
-    vb,
-    ut,
-    vt,
-    u,
-    v,
-    delp,
-    fx,
-    fy,
-    heat_source,
-    diss_est,
-    kinetic_energy_fraction_to_damp,
-    kstart,
-    nk,
-):
-    heat_source_from_vorticity_damping(
-        ub,
-        vb,
-        ut,
-        vt,
-        u,
-        v,
-        delp,
-        grid().rsin2,
-        grid().cosa_s,
-        grid().rdx,
-        grid().rdy,
-        heat_source,
-        diss_est,
-        kinetic_energy_fraction_to_damp,
-        int(spec.namelist.do_skeb),
-        origin=(grid().is_, grid().js, kstart),
-        domain=(grid().nic, grid().njc, nk),
-    )
-
-
-def set_low_kvals(col):
+def set_low_kvals(col, k):
```

Comment:
It's unclear to me why the code/functions you changed in this file do what they do in the way they do it, I'd appreciate any comments or docstrings you can add.

Reply:
I can add comments, but very likely this will all get removed in a near-future refactor in favor of hard coded specification of levels in stencils